core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -58,8 +56,7 @@
      */
     private final String identifier;
     private final List<File> composeFiles;
-    private final Set<String> spawnedContainerIds = new HashSet<>();
-    private final Set<String> spawnedNetworkIds = new HashSet<>();
```

Comment:
These two sets were never being written to, so they and all code that read them was redundant. Removed.